cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/BodyConsumerAdapter.java

```diff
@@ -173,6 +177,7 @@ public void run(DatasetContext context) throws Exception {
       }
     } catch (Throwable t) {
       responder.setTransactionFailureResponse(t);
+      LOG.debug("Exception in calling HttpContentConsumer.onError", t);
```

Comment:
I followed what we already do for the HttpContentProducer: https://github.com/caskdata/cdap/blob/develop/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/BodyProducerAdapter.java#L124